libs/groupSubscriptionManager.js

```diff
@@ -68,8 +71,7 @@ function chargeGroup (group, callback)
     // TODO should expire only in case of failed payment
     // otherwise retry
     if (response.AuthorizationDetails.AuthorizationStatus.State === 'Declined') {
-      cancelSubscription(group);
-      return;
+      return cancelSubscription(group);
```

Comment:
I think is a promise so the error will be catch in the catch statement in this chain.